tools/isolated-tests/src/RunTestsIsolatedCommand.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\Tools\IsolatedTests;
+
+use Contao\CoreBundle\Tests\PhpunitExtension\GlobalStateWatcher;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Process\Process;
+
+class RunTestsIsolatedCommand extends Command
```

Comment:
What's the difference to `phpunit --process-isolation`?

Reply:
That no state is shared at all. I tried `--process-isolation` but it delivered different results.

Reply:
Okay, then I think the command description or anything in this whole tool should say so :) So that other devs know why we have our own tool too :)

Reply:
Done in f75f185617c7decb606895815f09aa85f29625fe